src/components/Utils/orderResource.js

```diff
@@ -22,17 +22,15 @@ export const getFullOrderNumber = ({ poNumberPrefix = '', poNumberSuffix = '', p
 export const updateOrderResource = (order, mutator, changedProps) => {
   const clonedOrder = cloneDeep(order);
 
-  clonedOrder.poNumber = getFullOrderNumber(clonedOrder);
-
   Object.assign(clonedOrder, changedProps);
 
   return saveOrder(clonedOrder, mutator);
 };
 
-export const createOrderResource = (order, mutator) => {
-  const clonedOrder = cloneDeep(order);
+export const createOrEditOrderResource = (orderFormValues, mutator, changedProps) => {
+  const clonedOrder = cloneDeep(orderFormValues);
 
-  clonedOrder.poNumber = getFullOrderNumber(clonedOrder) || undefined;
```

Comment:
what is the point of moving this into Component?

Reply:
misunderstood your previous comment